scripts/train_lookup_lemmatizer.py

```diff
@@ -11,7 +13,7 @@
 from tqdm import tqdm
 
 app = typer.Typer()
-
+NUMBER_FORMAT: Pattern = re.compile(r"\d")
```

Comment:
This regexp looks a bit simplistic to me. Have you checked that it does not make any false positive lemma?